ci/stage-test-sandstone.jenkinsfile

```diff
@@ -1,17 +1,29 @@
 stage('test') {
-    env.no_cpu = sh(script:'nproc', returnStdout: true).trim()
-    timeout(time: 120, unit: 'MINUTES') {
-        sh '''
-            if [ "${no_cpu}" -gt 16 ]
-            then
-                cd CI-Examples/sandstone-50-bin
-                if test -n "$SGX"
-                then
+    if (env.no_cpu.toInteger() > 16)
+    {
+        try{
+            timeout(time: 120, unit: 'MINUTES') {
+                sh '''
+                    cd CI-Examples/sandstone-50-bin
                     make SGX=1 
```

Comment:
SGX is already part of env. It is okay if you only do make

Reply:
I have removed it now